motionplan/cBiRRT.go

```diff
@@ -59,11 +38,7 @@ type cbirrtOptions struct {
 // defaults, but can be tweaked if needed.
 func newCbirrtOptions(planOpts *plannerOptions, frame referenceframe.Frame) (*cbirrtOptions, error) {
 	algOpts := &cbirrtOptions{
-		FrameStep:       defaultFrameStep,
-		JointSolveDist:  defaultJointSolveDist,
 		SolutionsToSeed: defaultSolutionsToSeed,
-		IterBeforeRand:  defaultIterBeforeRand,
-		rrtOptions:      newRRTOptions(),
```

Comment:
lets just pass the frame into the newRRTOptions function and set the qstep inside there